Changelog.md

```diff
@@ -1,3 +1,8 @@
+Unreleased
+----------
+* [Enhancement] Retry `read_from_contentstore()`. Use `tenacity`'s
```

Comment:
I'd say making the code more robust is a bug fix, meaning I'd bump the patch number rather than the minor here. Would you agree?

Reply:
Ack, makes sense.